System.IO.FileSystem/File.cs

```diff
@@ -166,39 +139,88 @@ public static void Delete(string path)
         public static bool Exists(string path)
         {
             return ExistsNative(Path.GetDirectoryName(path), Path.GetFileName(path));
         }
 
+        /// <summary>
+        /// Gets the <see cref="FileAttributes"/> of the file on the path.
+        /// </summary>
+        /// <param name="path">The path to the file.</param>
+        /// <exception cref="IOException">File not found.</exception>
+        public static FileAttributes GetAttributes(string path)
+        {
+            // Adding this check because on my device the GetAttributesNative call throws an Exception instead of IOException if the file is not found
+            if (!Exists(path))
+            {
+                throw new IOException(string.Empty, (int)IOException.IOExceptionErrorCode.FileNotFound);
+            }
+
+            var attributes = GetAttributesNative(path);
+
+            if (attributes == 0xFF)
+            {
+                throw new IOException(string.Empty, (int)IOException.IOExceptionErrorCode.FileNotFound);
+            }
+
+            return (FileAttributes)attributes;
+        }
+
+        /// <summary>
+        /// Returns the date and time the specified file or directory was last written to.
+        /// </summary>
+        /// <param name="path">
+        /// The file or directory for which to obtain write date and time information.
+        /// </param>
+        /// <returns>
+        /// A <see cref="DateTime" /> structure set to the last write date and time for the specified file or directory.
+        /// </returns>
+        public static DateTime GetLastWriteTime(string path)
+        {
+            // Adding this check because on my device the GetLastWriteTimeNative call throws an Exception instead of IOException if the file is not found
+            if (!Exists(path))
+            {
+                throw new IOException(string.Empty, (int)IOException.IOExceptionErrorCode.FileNotFound);
+            }
+
+            return GetLastWriteTimeNative(path);
+        }
+
         /// <summary>
         /// Moves a specified file to a new location, providing the option to specify a new file name.
         /// </summary>
-        /// <param name="sourceFileName">The name of the file to move. Absolute path.</param>
+        /// <param name="sourceFileName">The name of the file to move. Must be an absolute path.</param>
         /// <param name="destFileName">The new path and name for the file.</param>
-        /// <exception cref="Exception">Source File not existing or Destination File already existing.</exception>
-        public static void Move(
-            string sourceFileName,
-            string destFileName)
+        public static void Move(string sourceFileName, string destFileName)
         {
-            // Src File must exists!
+            if (string.IsNullOrEmpty(sourceFileName))
+            {
+                throw new ArgumentException(nameof(sourceFileName));

```

Comment:
Same here. Please move these to the comments for exception which have been removed and they shouldn't.